content/packages/tornado-cash.md

```diff
@@ -1,210 +0,0 @@
----
```

Comment:
@sponnet I would not drop the file. I am OK will removing most of the content, but I'd add a message that it is no longer supported

Reply:
agreed - can you re-add that page then plz ?